openwisp_controller/config/tests/pytest.py

```diff
@@ -11,21 +11,22 @@
 from ..base.channels_consumer import BaseDeviceConsumer
 from .utils import CreateDeviceMixin
 
-Device = load_model('config', 'Device')
+Device = load_model("config", "Device")
```

Comment:
Can you please switch from ~`" "` (double quotes)~  to `' '` (single quotes) to maintain consistency with the rest of the codebase?